libs/secrets.js

```diff
@@ -0,0 +1,31 @@
+import AWS from 'aws-sdk';
```

Comment:
Should import our custom aws-sdk pkg to support future features

Reply:
Don't se why we should use the aws-sdk pkg defined in /libs, it currently does nothing more than returning the aws import? Or for that manner why, we have added a wrapper somewhere along the line?

Reply:
Just don't get the reason for having a wrapper, and i can see know that we have been using it mixed with normal aws-sdk imports. 

So some files are importing from 'aws-sdk' and others from './aws-sdk'

Reply:
remove the wrapper for now then